docs/project/olmv1_limitations.md

```diff
@@ -3,23 +3,22 @@ hide:
   - toc
 ---
 
-## OLM v0 Extension Support
+## Content Support
 
-Currently, OLM v1 supports installing cluster extensions that meet the following criteria:
-
-* The extension must support installation via the `AllNamespaces` install mode.
-* The extension must not use webhooks.
-* The extension must not declare dependencies using any of the following file-based catalog properties:
+Currently, OLM v1 supports only supports installing operators packaged in [OLM v0 bundles](https://olm.operatorframework.io/docs/tasks/creating-operator-bundle/). 
```

Comment:
There is a repetition of  `supports` word in ` supports only supports installing operators`